examples/gno.land/r/matijamarjanovic/snakeart/snakeart.gno

```diff
@@ -0,0 +1,78 @@
+package snake
```

Comment:
I suggest you separate this out in a different PR, as the config & home packages will need to be reworked quite a bit.

Reply:
I don't think it's worthy of saving so I will just leave it out